components/provisioner/pkg/gqlschema/schema.graphql

```diff
@@ -212,6 +213,7 @@ input ComponentConfigurationInput {
     namespace: String!                    # Namespace to which component should be installed
     configuration: [ConfigEntryInput]     # Component specific configuration
     sourceURL: String                     # Custom URL for the source files of the given component
+    onConflict: String                            # Defines merging strategy if conflicts occur for component overrides
```

Comment:
onConflict field could be defined as Enum type (similar to e.g. KymaProfile). It makes sense as there are two possible values and you get validation for free. Here is the error message I got whn I specified incorrect KymaProfile value: "Expected type KymaProfile, found Evaluationdd. Did you mean the enum value Evaluation?"